scripts/zones/RuLude_Gardens/Zone.lua

```diff
@@ -126,5 +136,19 @@ function onEventFinish(player, csid, option)
     elseif csid == 143 then
         player:completeQuest(JEUNO, tpz.quest.id.jeuno.STORMS_OF_FATE)
         player:setCharVar('StormsOfFate', 0)
+		player:setCharVar("StormsOfFateWait",getMidnight())
+    elseif csid == 161 then
+		npcUtil.giveKeyItem (player, tpz.ki.NOTE_WRITTEN_BY_ESHANTARL)
+        player:addQuest(JEUNO,tpz.quest.id.jeuno.SHADOWS_OF_THE_DEPARTED)
+    elseif csid == 162 then
+        player:completeQuest(JEUNO,tpz.quest.id.jeuno.SHADOWS_OF_THE_DEPARTED)
+        player:delKeyItem(tpz.ki.PROMYVION_HOLLA_SLIVER)
+        player:delKeyItem(tpz.ki.PROMYVION_DEM_SLIVER)
+        player:delKeyItem(tpz.ki.PROMYVION_MEA_SLIVER)
+        player:setLocalVar('ANZONE', 1)
+        player:setCharVar("ApocNighWait",getMidnight())	
```

Comment:
Think you'll want [`getVanaMidnight` (next game day)](https://github.com/project-topaz/topaz/blob/release/scripts/globals/common.lua#L73-L78) here instead of `getMidnight` (server midnight)